packages/terra-functional-testing/src/commands/utils/GitHubRepo.js

```diff
@@ -0,0 +1,148 @@
+/**
+ * This module provides a simplified interface to github repos.
+ *
+ * It's basically a wrapper around octokit. The purpose here is to avoid people
+ * re-implememnting octokit w/ auth and error handling in many places.
+ *
+ * The JSON datatypes are filtered versions of the JSON representations you
+ * can read about at https://docs.github.com/en/rest.
+ */
+
+const { Octokit } = require('octokit');
+
+/**
+ * Information about a PR.
+ * @typedef {Object} Pull
+ * @property {number} number
+ * @property {Object} head - The head branch
+ * @property {string} head.ref
+ * @property {Object} base - The base branch
+ * @property {string} base.ref
+ */
+
+/**
+ * Create a simplified view of a PR.
+ *
+ * @param {object} json A PR from the ReST API.
+ * @returns {Pull}
+ */
+function filterPull(json) {
```

Comment:
nit: would `filterPullRequest` be a better name? `filterPull` is confusing out of context. `filterPR` also works if you want to minimize characters.

Reply:
Fixed in a7cf278f59aa9da516cd4c5da20645f2c7d162cf.